backend/app/services/skill_binding_service.py

```
@@ -219,13 +219,50 @@ def list_group_skill_ids(
+        group_namespaces: list[str],
+    ) -> set[int]:
+        """Return active Skill ids bound to already-authorized groups."""
+        if not group_namespaces:
+            return set()
+
+        bindings = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == SKILL_BINDING_KIND,
+                Kind.namespace.in_(group_namespaces),
+                Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)